### ISSUE TITLE
Rewards on x ..

### ISSUE THREAD

**Issue by rosiejane** [PyvesB/advanced-achievements#977]
Hello. @PyvesB 

Please can we have the option to reward players for x amount of achievements completed.

Lets say i want to give players a permission or crate keys when they complete 50, 100, 300 achievements.

The "AdvancementsCompleted" does not work like this from what i have tested.

Thank you

**Comment by PyvesB** [comment]
@rosiejane this is the third issue from you that I'm having to close as duplicate (this one duplicating multiple recent issues that can easily be found) and the fifth one that does not follow any of the issue templates. There has been no evidence of you putting any effort or doing any research before opening new issues or unnecessarily pinging me on existing ones (e.g. https://github.com/PyvesB/advanced-achievements/issues/781#issuecomment-786971647). This is an open-source project, if you're not willing to engage or contribute in a useful and constructive manner, please refrain from asking for people's help.

**Comment by rosiejane** [comment]
> 
> 
> @rosiejane this is the third issue from you that I'm having to close as duplicate (this one duplicating multiple recent issues that can easily be found) and the fifth one that does not follow any of the issue templates. There has been no evidence of you putting any effort or doing any research before opening new issues or unnecessarily pinging me on existing ones (e.g. [#781 (comment)](https://github.com/PyvesB/advanced-achievements/issues/781#issuecomment-786971647)). This is an open-source project, if you're not willing to engage or contribute in a useful and constructive manner, please refrain from posting any further messages in this issue tracker.

Absolutely no need to be rude. You still did not answer the question and what you linked to above (#781) is completely different to what i have asked here...

**Comment by PyvesB** [comment]
With such a response, you've demonstrated that you've not actually properly read nor understood my response. You're visibly confused about what I linked as an example of your past interactions on the repository and are surprisingly expecting it to be the answer to this unrelated duplicate issue. You're claiming that a mostly descriptive and perfectly polite message is somehow rude and you're still refusing to even invest the tiny bit of effort into searching existing issues, effectively demanding help to be handed on a silver platter for free.

This is frankly offensive, especially if one considers all the time that I've dedicated to helping you in the past and the fact that you've not given back to the project in any way or form.
